less/slider.less

```diff
@@ -64,6 +65,28 @@
   // --------------------------------------------------
   &__item {
     position: relative;
+
+    input {
+      width: 100%;
+    }
```

Comment:
@chris-steele Since the slider _bar_ is clickable, it should probably have a pointer cursor style.
```suggestion
      &:not([disabled]) {
        cursor: pointer;
      }
    }
```